spec.bs

```diff
@@ -5662,6 +5603,113 @@ A <dfn>per signals url bid generator</dfn> is an [=ordered map=] whose [=map/key
 representing [=interest group/joining origins=], and whose [=map/values=] are [=lists=] of
 [=interest groups=].
 
+<h3 id="script-fetcher-section">Script fetcher</h3>
+
+A <dfn>script fetcher</dfn> helps manage asynchronous fetching of scripts and handling of their
+headers. It's a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="script fetcher">
+  : <dfn>script body</dfn>
+  :: A [=byte sequence=], null or failure. Initially null. The body of the script.

```

Comment:
Comma after null.  At least the Oxford comma seems to be pretty universal, at this point?

Reply:
Done